CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## 0.6.0+1
```

Comment:
New features/APIs – 0.6.1
Breaking changes - 0.7.0

Looks like there are some of both?

Reply:
Yes, this change grew over time and I never I bumped the version number.